src/Microsoft.DotNet.ImageBuilder/src/Commands/GitOptions.cs

```diff
@@ -125,7 +126,13 @@ public GitOptionsBuilder WithGitHubAuth(string? description = null, bool isRequi
                 nameof(GitHubAuthOptions.ClientId),
                 "GitHub Client ID for GitHub App authentication");
 
-            _options.AddRange([tokenOption, privateKeyOption, clientIdOption]);
+            const string InstallationIdAlias = "gh-app-installation-id";
+            var installationIdOption = CreateOption<long?>(
+                InstallationIdAlias,
+                nameof(GitHubAuthOptions.InstallationId),
+                "GitHub App installation ID to use (optional; if not provided, it will be discovered automatically)");
```

Comment:
This could describe how the option is required if the app has more than one installation.

```suggestion
                "GitHub App installation ID to use (only required if app has more than one installation)");
```